SparkPost/2web2ui
# SA-1425: Add Rejection Reasons Tab

### What Changed 
 - Added rejection reasons tab to report builder

### How To Test
- Go to `reports/summary` with hibana enabled. Check that all only these metrics show the rejection tab. 
    - Rejected
    - Generation Rejection
    - Generation Failure
    - Policy Rejection
 - Verify that the tab and table appears.
 - Also check that I changed any text correctly when I copy+pasted
